js/components/controlBar.js

```diff
@@ -288,6 +280,23 @@ var ControlBar = React.createClass({
     }
   },
 
+  /**
+   * @description the function closes popovers (closedCaptionPopover, videoQualityPopover, multiAudioPopover);
+   * if the parameter specifies the name of the popover, then its state does not change
+   * @param popoverName {string} - the name of the popover that does not need to be closed
+   * @private
+   */
+  closeOtherPopovers: function(popoverName) {
+    var popoversNameList = [CONSTANTS.MENU_OPTIONS.CLOSED_CAPTIONS, CONSTANTS.MENU_OPTIONS.VIDEO_QUALITY, CONSTANTS.MENU_OPTIONS.MULTI_AUDIO];
+    for (var index = 0; index < popoversNameList.length; index++) {
```

Comment:
Thanks for your changes!
It seems that we need to execute `this.closePopover` for every name in `popoversNameList`, could we possibly rewrite it like
```
popoversNameList.forEach(function (popoverName) { 
  this.closePropover(popoverName);
}, this); ?
 
```
or to be more clean 
```
popoversNameList.forEach(this.closePropover, this); ?
```

Reply:
Let's save "for".